src/Post.php

```diff
@@ -75,10 +95,16 @@ private function get_query_filter_name() {
 	 */
 	public function endpoint_args() {
 		return [
+			'id' => [
+				'default' => false,
```

Comment:
I think default id should be 0 and default slug should be empty string instead of `false` and I think we just use absint instead, and if some one uses -66 that is a problem of how they use the API, [10up recommends](https://10up.github.io/Engineering-Best-Practices/php/) to use absint for sanitization of IDs and makes code easier as well.